src/Pages/Cart/Cart.js

```diff
@@ -0,0 +1,316 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import CartItem from "./CartItem";
+import { NumberWithCommas } from "./NumberWithCommas";
+import config from "../../config";
+
+function Cart() {
+  const [productPrice, setProductPrice] = useState(0);
+  const [totalPrice, setTotalPrice] = useState(0);
+  const [shipping, setShipping] = useState(0);
+  const [Ischecked, setIschecked] = useState(true);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    const headers = {
+      Authorization:
+        "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZGVudGlmaWVyIjoid2xkdXMxMjMifQ.oNOUNs-6XnGUw3gwgQC3M9EOPugekcyVUx2VNdLkJ2g",
+    };
+
+    fetch(`${config.API}/order/cart`, { headers })
+      .then((response) => response.json())
+      .then((res) => {
+        setData(res.product_list);
+        GettotalPrice(res.product_list);
+      });
+  }, []);
+
+  const GettotalPrice = (data) => {
+    let total = 0;
+    let shippingFee = 0;
+
+    for (let i = 0; i < data.length; i++) {
+      total += data[i].price * data[i].quantity;
+    }
```

Comment:
```suggestion
    data.reduce((acc, cur) => {
      return acc + cur.price * cur.quantity
    }, 0)
```
리듀스를 사용하면 간단하게 배열 메소드 만으로 총합을 구할 수 있습니다